README.md

```diff
@@ -824,6 +825,228 @@ and Equinox will supply the _initial_ value for the `project` function to render
+
+With the Equinox `type Decider`, the typical `decide` signature used with the `Transact` API has the following signature:
+
+      context -> inputsAndOrCommand -> 'State -> Event list
+
+> NOTE: There are more advanced forms that allow the `decide` function to be `Async` and/or to also return a `'result`, which will be yielded to the caller driving the Decision as the return value of the `Transact` function.
+
+#### So what kind of a thing is a Decider then?
+
+Is it a marketing term? Jérémie's way of explaining an app?
+
+I'd present the fact that Equinox:
+- was initially generalised and extracted from working code using ESDB in (by most measures) a successful startup written in a mix of F# and C# by the usual mix of devs, experience levels, awareness of event sourcing patterns and language/domain backgrounds
+- for a long time only had its MemoryStore as the thing to force it to be store agnostic
+- did not fundamentally change to add idiomatic support for a Document database (CosmosDB)
+- will not fundamentally change to add idiomatic support for DynamoDB
+- can and has been used at every stage in an e-commerce pipeline
+- is presently aligning pretty neatly with diverse domains without any changes/extensions, both for me and others
+
+... as evidence for Decider being *a pattern language* (analogous to how event sourcing and many modern event sourced UI paradigms share a lot of common patterns).
+
+#### ... about the process of making _decisions_
+
+Finally, I'd say that a key thing the Decider concept brings is a richer way of looking at event sourcing than the typical event sourcing 101 examples you might see:
+- de-emphasizing one-way things that map commands to events without deduplicating and/or yielding a result (not saying that you shouldn't do the simplest thing -- you absolutely should)
+- de-emphasizing all projection handlers only ever just sitting there looking for `MyThingCreated` and doing an `INSERT` with a try/catch for duplicate inserts as a way to make it idempotent (and every stream design requiring a Created event as a way to enable that)
+
+The missing part beyond that basic anemic stuff is where the value lies:
+- any interesting system *makes _decisions_*:
+- a decision can yield a result alongside the events that are needed to manifest the state change
+- any decision process can and should consider [idempotency](https://en.wikipedia.org/wiki/Idempotence) - if you initiate a process/request something, a retry can't be a special case you don't worry about, considering it is a fundamental baseline requirement and thinking tool
+- it can let you drive a set of reactions in a fault-tolerant and scalable (both perf/system size, and management/separation of complexity)  manner
+
+Quite frequently, a Decider will also internally be a Process Manager, encapsulting a state machine.
```

Comment:
**QUESTION:** Could you elaborate on that part? Do you mean that grouping multiple decisions will create effectively Process manager?

Reply:
Elaborated, wil push later. No, this is more an extension of not having anemic aggregates - there needs to be a reason for a thing to exist, invariants being enforced. Not everytihng needs that, but something does. wrt this overall question, you're right that there is a conflict/concern - if you have 2 deciders with 2 sets of rules, those can be independent, but if e.g. you do conflict checking based on expected version oinly, thy confkict and trigger retries for no reason. Not sure where that would be good to mention in the flow, and this is all very long altready.